modules/core/05-port/types/module.go

```diff
@@ -110,8 +111,12 @@ type ICS4Wrapper interface {
 	SendPacket(
 		ctx sdk.Context,
 		chanCap *capabilitytypes.Capability,
-		packet exported.PacketI,
-	) error
+		sourcePort string,
+		sourceChannel string,
+		timeoutHeight clienttypes.Height,
+		timeoutTimestamp uint64,
+		data []byte,
+	) (sequence uint64, err error)
```

Comment:
Should we leave these as unnamed return args as they are in other methods such as `GetAppVersion`?

Reply:
It is good as is in my opinion. OnChanOpenInit has named returns. GetAppVersion probably should have had a named return

Reply:
Maybe we can make them all consistent in housekeeping followup.